Example/Messaging/Tests/FIRMessagingContextManagerServiceTest.m

```diff
@@ -83,8 +83,10 @@ - (void)testMessageWithFutureStartTime {
   XCTAssertTrue([FIRMessagingContextManagerService handleContextManagerMessage:message]);
 
   XCTAssertEqual(self.scheduledLocalNotifications.count, 1);
-
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   UILocalNotification *notification = [self.scheduledLocalNotifications firstObject];
+#pragma pop
```

Comment:
Should these all be `#pragma clang diagnostic pop`?

Reply:
Yep. And a few others. Thanks!